dbms/src/Common/FmtUtils.h

```diff
@@ -14,35 +14,43 @@
 
 #pragma once
 
+#include <fmt/compile.h>
 #include <fmt/format.h>
 
 namespace DB
 {
+
 class FmtBuffer
 {
 public:
     FmtBuffer() = default;
 
-    template <typename... Args>
-    FmtBuffer & fmtAppend(std::string_view fmt, Args &&... args)
+    template <typename FmtStr, typename... Args>
+    inline FmtBuffer & fmtAppend(FmtStr fmt, Args &&... args)
```

Comment:
methods defined inside of the class body is implicitly inlined.

Reply:
Thanks! Lessons learnt 👍